beetsplug/lyrics.py

```diff
@@ -401,7 +401,12 @@ def _scrape_lyrics_from_html(self, html):
         # all of the lyrics can be found already correctly formatted
         # Sometimes, though, it packages the lyrics into separate divs, most
         # likely for easier ad placement
-        lyrics_div = soup.find("div", class_="lyrics")
+
+        lyrics_div = soup.find("div", {"data-lyrics-container": True})
+
+        for br in lyrics_div.find_all("br"):
```

Comment:
Looks like some tests are failing here because `lyrics_div` may be `None`. Maybe this just needs to go inside the `if` block below?